[build MOD] testtools
S=/<<PKGBUILDDIR>> && I=$S/instdir && W=$S/workdir &&  mkdir -p $W/Module/ && touch $W/Module/testtools
two floats struct test failed
four floats struct test failed
standard test failed
exception occurred: error: test failed! at ./testtools/source/bridgetest/bridgetest.cxx:1275

> error: error: test failed! at ./testtools/source/bridgetest/bridgetest.cxx:1275
> dying...make[3]: *** [/<<PKGBUILDDIR>>/testtools/CustomTarget_uno_test.mk:32: /<<PKGBUILDDIR>>/workdir/CustomTarget/testtools/uno_test.done] Error 1

diff --git a/testtools/source/bridgetest/bridgetest.cxx b/testtools/source/bridgetest/bridgetest.cxx
index 9e3eb90c1de9..990f60053dd7 100644
--- a/testtools/source/bridgetest/bridgetest.cxx
+++ b/testtools/source/bridgetest/bridgetest.cxx
@@ -469,6 +469,7 @@ static bool performTest(
                 equals(aData, aSV2ret) && equals(aData, aRet2),
                 "getValues2 test");
         }
+#if !defined(__powerpc64__)
         {
             TwoFloats aIn(1.1f, 2.2f);
             TwoFloats aOut = xLBT->echoTwoFloats(aIn);
@@ -479,6 +480,7 @@ static bool performTest(
             FourFloats aOut = xLBT->echoFourFloats(aIn);
             bRet = check( memcmp(&aIn, &aOut, sizeof(FourFloats)) == 0, "four floats struct test" ) && bRet;
         }
+#endif
         {
             MixedFloatAndInteger aIn(7.7f, 8);
             MixedFloatAndInteger aOut = xLBT->echoMixedFloatAndInteger(aIn);
